okok/node-api/api.js

```diff
@@ -62,179 +62,212 @@ const createUser = async (request, response) => {
-                                            </div>
-                                          </td>
-                                        </tr>
-                                        <tr>
-                                          <td align='center' style='font-size:0px;padding:10px 25px;word-break:break-word;' vertical-align='middle'>
-                                            <table border='0' cellpadding='0' cellspacing='0' role='presentation' style='border-collapse:separate;line-height:100%;'>
-                                              <tr>
-                                                <td align='center' bgcolor='#ff6f6f' role='presentation' style='background-color:#ff6f6f;border:none;border-radius:5px;cursor:auto;padding:10px 25px;' valign='middle'>
-                                                  <a href='http://bond-nu.vercel.app/accept?bondkey=`+encryptedEmail+`' style='background:#ff6f6f;color:#ffffff;font-family:Oxygen, Helvetica neue, sans-serif;font-size:14px;font-weight:400;line-height:21px;margin:0;text-decoration:none;text-transform:none;' target='_blank'>
-                                                    Bond now!
-                                                  </a>
-                                                </td>
-                                              </tr>
-                                            </table>
-                                          </td>
-                                        </tr>
-                                      </table>
-                                    </td>
-                                  </tr>
-                                </tbody>
-                              </table>
-                            </div>
-                            <!--[if mso | IE]>
-    </td></tr></table>
-    <![endif]-->
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table>
-                  </div>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-      </body>
-    </html>
-    `
-    return templete
-  }
-  // end of function that holds the formatted html invite
+                              </td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </div>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </body>
+        </html>
+        `
+      
+        return templete
+      }
+      // end of function that holds the formatted html invite
 
 
-  // encripting the sender's email to use in the invite
-  const cryptr = new Cryptr('bondkey');
-  const encryptedEmail = cryptr.encrypt(request.params.email);
+      // encripting the sender's email to use in the invite
+      const cryptr = new Cryptr('bondkey');
+      const encryptedEmail = cryptr.encrypt(receiver);
```

Comment:
Hey please, could you encrypt the sender's mail instead? Because we will need that later on remember?

Reply:
and also please could you rename `encryptedEmail` to `senderEncryptedEmail ` for better understanding?
Thanks

Reply:
cool2